nservicebus/gateway/scale-out.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gateway And Scale-Out
+summary: Explanation of how the Gateway handles Scale-Out
+tags: []
+related:
+ - samples/gateway
+---
+
+The gateway is designed to run on a single instance for each logical endpoint. A scaled out endpoint needs one instance to receive all the incoming HTTP gateway traffic for that endpoint. There are different strategies to achieve this based on the transport and the scale-out mechanism utilized:
```

Comment:
How about `Due to specifics of the protocol used the gateway is designed to run on at most one instance of each endpoint`. Depending on the transport there are different strategies for designating a gatway-hosting endpoint.